ethereumj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -671,6 +674,61 @@ else if (oldValue != null && newValue.isZero()) {
                     program.step();
                 }
                 break;
+                case SHL: {
+                    DataWord word1 = program.stackPop();
+                    DataWord word2 = program.stackPop();
+                    final DataWord result;
+                    if (word1.value().compareTo(BigInteger.valueOf(256)) < 0) {
```

Comment:
This and following checks (in other similar instructions) is better to move to `DataWord` class cause they also make sense for `DataWord` itself